course_catalog/admin.py

```diff
@@ -122,6 +134,16 @@ class ProgramAdmin(admin.ModelAdmin):
     inlines = [ProgramItemInline, LearningResourceRunInline]
 
 
+class StaffListAdmin(admin.ModelAdmin):
```

Comment:
Thanks for adding this. I think it's good since we have it for the other catalog models, too.